src/Sylius/Behat/Client/ApiPlatformSecurityClient.php

```diff
@@ -24,18 +24,14 @@ final class ApiPlatformSecurityClient implements ApiSecurityClientInterface
     /** @var array */
     private $request = [];
 
-    /** @var string */
-    private $section;
-
-    public function __construct(AbstractBrowser $client, string $section)
+    public function __construct(AbstractBrowser $client)
     {
         $this->client = $client;
-        $this->section = $section;
     }
 
     public function prepareLoginRequest(): void
     {
-        $this->request['url'] = sprintf('/new-api/%s/authentication-token', $this->section);
+        $this->request['url'] = sprintf('/new-api/authentication-token');
```

Comment:
I believe it should be adjusted to [this](https://github.com/Sylius/Sylius/pull/11397/files#diff-985d14deddf2a290f44196618feab0f2R78) and [this](https://github.com/Sylius/Sylius/pull/11397/files#diff-985d14deddf2a290f44196618feab0f2R63)